coalib/settings/Setting.py

```diff
@@ -13,6 +14,22 @@ def path_list(obj, *args, **kwargs):
     return obj.__path_list__(*args, **kwargs)
 
 
+def glob(obj, *args, **kwargs):
+    """
+    Creates a path in which all special glob characters in all the
+    parent directories in the given setting are properly escaped.
+    """
+    return obj.__glob__(*args, **kwargs)
+
+
+def glob_list(obj, *args, **kwargs):
+    """
+    Creates a list of paths in which all special glob characters in all the
```

Comment:
But I think we can introduce other types of sequences too, but I fail to see the purpose: this is meant to be a function that basically returns a list of full paths (glob escaped) for the given list of globs.